spiceaidocs/docs/api/odbc/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 'ODBC: Open Database Connectivity'
+sidebar_label: 'ODBC'
+sidebar_position: 2
+description: 'ODBC API Documentation'
+pagination_prev: null
+pagination_next: null
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+[ODBC](https://learn.microsoft.com/en-us/sql/odbc/microsoft-open-database-connectivity-odbc) (Open Database Connectivity) is a low-level, high-performance interface that is designed specifically for relational data stores as a standard way to connect to, and interact with a database.
+
+Spice supports ODBC clients through an ODBC driver implementation based on the [Flight SQL](https://arrow.apache.org/docs/format/FlightSql.html) protocol. This enables ODBC-compatible applications to connect to Spice, execute queries, and retrieve data.
+
+:::warning[Limitations]
+
+1. ODBC support is currently in alpha, and not all functionality is supported
+1. The Arrow Flight SQL ODBC driver is not available for 32-bit Windows versions
+1. The Arrow Flight SQL ODBC driver is not supported on the Apple ARM architecture
+
+:::
+
+## Install and configure the Flight SQL ODBC driver
+
+### Download and install the Flight SQL ODBC driver
+
+- Download and install the driver from the [ODBC driver download page](https://www.dremio.com/drivers/odbc/)
+  - [Windows instructions](https://docs.dremio.com/current/sonar/client-applications/drivers/arrow-flight-sql-odbc-driver/#downloading-and-installing-on-windows)
+  - [Linux](https://docs.dremio.com/current/sonar/client-applications/drivers/arrow-flight-sql-odbc-driver/#downloading-and-installing-on-linux)
+  - [macOS instructions](https://docs.dremio.com/current/sonar/client-applications/drivers/arrow-flight-sql-odbc-driver/#downloading-and-installing-on-macos)
+
+### Configure Flight SQL ODBC driver
+
+<Tabs>
+  <TabItem value="default" label="Windows" default>
+    - Open **Start Menu** -> **Windows Administrative Tools** -> click **ODBC Data Sources (64-bit)**
+    - In the **ODBC Data Source Administrator (64-bit)** dialog, click **System DSN**
```

Comment:
Can we get a screenshot of the ODBC Data Source Administrator screen?

Reply:
Added for Windows and macOS 👍